BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -138,9 +138,12 @@ def create_stochastic_schedules(args:)
       all_simulated_values << Matrix[*simulated_values]
     end
     # shape of all_simulated_values is [2, 35040, 7] i.e. (geometry_num_occupants, period_in_a_year, number_of_states)
-    plugload_weekday_sch = Schedule.validate_values(Constants.PlugLoadsOtherWeekdayFractions, 24, 'weekday')
-    plugload_weekend_sch = Schedule.validate_values(Constants.PlugLoadsOtherWeekendFractions, 24, 'weekend')
-    plugload_monthly_multiplier = Schedule.validate_values(Constants.PlugLoadsOtherMonthlyMultipliers, 12, 'monthly')
+    plugload_other_weekday_sch = Schedule.validate_values(Constants.PlugLoadsOtherWeekdayFractions, 24, 'weekday')
+    plugload_other_weekend_sch = Schedule.validate_values(Constants.PlugLoadsOtherWeekendFractions, 24, 'weekend')
+    plugload_other_monthly_multiplier = Schedule.validate_values(Constants.PlugLoadsOtherMonthlyMultipliers, 12, 'monthly')
```

Comment:
Should we update these to follow Table C.3(1) of ANSI/RESNET/ICC 301-2022 Addendum C?

Reply:
I'll show how much difference we're talking here.

Reply:
The CI only really shows annual results; this makes a huge change to the timeseries results. Are you sure you want to do this?

Before:
![image](https://github.com/NREL/OpenStudio-HPXML/assets/5861765/e1d0224a-ab6d-42bf-b8fb-affd4a6f7a45)

After:
![image](https://github.com/NREL/OpenStudio-HPXML/assets/5861765/a3117386-fe5b-4f2f-a3ac-fe745afacb89)

Reply:
Right, for Other and TV we're moving to constant monthly multipliers.